src/coreclr/vm/jitinterface.cpp

```diff
@@ -14811,3 +14811,11 @@ ULONG EECodeInfo::GetFrameOffsetFromUnwindInfo()
     return frameOffset;
 }
 #endif // defined(TARGET_AMD64)
+
+#ifdef TARGET_WASM
+ULONG EECodeInfo::GetFixedStackSize()
```

Comment:
This should not be needed, as this function is ever called on amd64 and x86.

Reply:
I think I was hitting this before I disabled FEATURE_METADATA_UPDATER on wasm.

I removed the change.